tox.ini

```diff
@@ -7,5 +7,6 @@ deps =
     pytest
     pytest-cov
     mock
+    hubstorage
```

Comment:
@vshlapakov you need to add py34 and py35 to tox config now and ensure tests pass for all environments